python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -336,6 +331,17 @@ def set_gpu_ids():
                         worker_id, set_gpu_ids))
         ray.get(futures)
 
+    def _create_local_rank_map(self) -> Dict:
+        """Create mapping from worker world_rank to local_rank."""
```

Comment:
nit: Maybe include an example in the doc to explain the expected functionality?

```
Worker 0: 0.0.0.0
Worker 1: 0.0.0.0
Worker 2: 0.0.0.1
Worker 3: 0.0.0.0
Worker 4: 0.0.0.1

Workers 0, 1, 3 are on 0.0.0.0.
Workers 2, 4 are on 0.0.0.1.

Expected Output:
{
    0 -> 0, 
    1 -> 1, 
    2 -> 0,
    3 -> 2, 
    4 -> 1
}

```

Reply:
Added!